docs/07-how_to_run_a_node/03-run_using_docker/04-run-validator-node.md

```diff
@@ -0,0 +1,212 @@
+
+```bash
+docker compose -f /home/your_user/compose-zkverify-simplified/deployments/validator-node/testnet/docker-compose.yml up -d
+```
+
+*and you are done!* This allows you to start the node in background, for checking it is running properly you can type:
+
+```bash
+docker container ls
+```
+
+and you should get something similar to:
+
+```bash
+CONTAINER ID   IMAGE                            COMMAND                CREATED              STATUS              NAMES
+ca4bdf2c6f05   horizenlabs/zkverify:0.2.0-rc1   "/app/entrypoint.sh"   About a minute ago   Up About a minute   validator-node
+```
+
+showing your node has started correctly.
+
+Up to here your validator node is running properly but, in order to make it eligible for authoring new blocks, you need to follow through with a few additional steps.
+
+## Next Steps
+
+In this section you can learn how to register a new validator on the blockchain. The operations described below must be performed just once and they consist in the submission of some extrinsics (transactions, in Substrate terminology) resulting eventually in your node being able to author new blocks and consequently earn new tokens through staking mechanism. Note that since you are going to submit extrinsics which changes the blockchain state, you need sufficient funds in the account (uniquely identified by your secret phrase) associated with your validator so that you can pay transaction fees; if that's not your case, it is mandatory you send some tokens to it.
+
+For security reasons your validator node does not expose an RPC interface but you need a user friendly way for submitting the extrinsics, so the first thing to do is to temporarily run an additional RPC node (refer to [this page](./02-run-rpc-node.md)) which you can shutdown once you have completed everything.
+
+Make sure both your validator node as well as your RPC node are running using command:
+
+```bash
+docker container ls
+```
+
+and you should get something similar to:
+
+```bash
+CONTAINER ID   IMAGE                         COMMAND                CREATED              STATUS              NAMES
+096d0c5d1117   horizenlabs/zkverify:latest   "/app/entrypoint.sh"   About a minute ago   Up About a minute   rpc-node
+b693d6143f0d   horizenlabs/zkverify:latest   "/app/entrypoint.sh"   10 minutes ago       Up 10 minute        validator-node
+
+```
+
+Then you can connect to the RPC node using PolkadotJS (refer to [this section](./02-run-rpc-node.md#Explore-and-Interact-with-the-Node) for a brief walkthrough) and start submitting RPC commands and extrinsics.
+
+In order to use PolkadotJS with your validator account, you need to import it within the application.
+
+From here on you can choose any the extrinsic you submit with PolkadotJS to use your validator account (even if you are submitting them with your rpc node, doesn't matter since you imported your validator account), hence transaction fees will be deducted from its balance.
```

Comment:
Guide on this page doesn't state anywhere how to import your validator account, or how to get to your seed phrase. Information on how to do this has to be gathered from multiple other pages in the docs like preliminaries and run-rpc-node.

Reply:
The import or generation of the secret-phrase is handled by `init.sh` script and there's a reference to that step 
(_- secret phrase (`secret_phrase.dat` file): you can import an already existing secret phrase or let the script to randomly generate one for you (refer to [this page](./01-preliminaries.md) for secret phrases)._). So the docker-side of the guide should be covered.
For the binaries-side of the guide I have added a section describing secret phrase handling here (`./07-how_to_run_a_node/04-run_using_binaries/04-run-validator-node.md#prepare-and-run`).